rpc-server/src/dispatchers/wallet.rs

```diff
@@ -114,7 +114,7 @@ impl WalletInterface for WalletDispatcher {
 
         self.unlocked_wallets.write().insert(unlocked_account);
 
-        Ok(())
+        Ok(true)
```

Comment:
Why does it now return a bool? It's only ever set to true, isn't it?

Reply:
It gives a json parsing error if it returns nothing/null. My understanding is that this issue was not detected previously because only Soeren is using the rpc client and he is using the javascript one.